packages/altair-api/src/auth/mocks/express.mock.ts

```diff
@@ -0,0 +1,9 @@
+import { Request, Response } from 'express';
+
+export function mockRequest(props?: object): Request {
```

Comment:
```suggestion
export function mockRequest(props?: Partial<Request>): Request {
```

the props should be valid fields in request right? Same treatment for response below.